pallets/connectors/src/message.rs

```diff
@@ -0,0 +1,32 @@
+use crate::*;
+
+#[derive(Decode, Clone, PartialEq, TypeInfo)]
+#[cfg_attr(feature = "std", derive(Debug))]
+pub enum Message<PoolId> {
+	AddPool { pool_id: PoolId }, // More to come...
```

Comment:
We need to make sure the indices of this enum matches the indices in the EVM implementation: https://github.com/centrifuge/connectors/blob/main/src/Messages.sol#L10 There it starts with an `Invalid` variant (copied from the Nomad sample implementation, my assumption is that the reason for this is that a message that is entirely zeros (0x000000...) is not considered a valid message. If the first message is `AddPool` and you add a pool ID of 0, then the full message would actually become 0x00000...).